gamedb.yaml

```diff
@@ -5493,3 +5493,745 @@
+
+- name: "DOOM 3: BFG Edition"
+  platform: gog
+  id: 1135892318
+  banner: steam:208200
+  status: unsupported
+
+- name: "Ghost Master"
+  platform: gog
+  id: 1207658687
+  banner: steam:6200
+  status: unsupported
+
+- name: "Greak: Memories of Azur"
+  platform: gog
+  id: 2142923325
+  banner: steam:1311070
+  status: unsupported
+
+- name: "Hellpoint"
+  platform: gog
+  id: 1503950763
+  banner: steam:628670
+  status: unsupported
+
+- name: "I Have No Mouth And I Must Scream"
+  platform: gog
+  id: 1207659593
+  banner: steam:245390
+  status: unsupported
+
+- name: "Janosik - Highlander Precision Platformer"
+  platform: gog
+  id: 1813565692
+  banner: steam:1412170
+  status: unsupported
+
+- name: "Neverwinter Nights Diamond"
+  platform: gog
+  id: 1207658890
+  banner: https://cdn2.steamgriddb.com/grid/7012fda2190dc3120e31b667cf9e2c06.jpg
+  status: unsupported
+
+- name: "Neverwinter Nights: Enhanced Edition"
+  platform: gog
+  id: 1097893768
+  banner: steam:704450
+  status: unsupported
+
+- name: "Outcast 1.1"
+  platform: gog
+  id: 1207658853
+  banner: steam:336610
+  status: unsupported
+
+- name: "Sanitarium"
+  platform: gog
+  id: 1207658811
+  banner: steam:284050
+  status: unsupported
+
+- name: "STASIS"
+  platform: gog
+  id: 1439548178
+  banner: steam:380150
+  status: unsupported
+
+- name: "Shadowrun Hong Kong - Extended Edition"
+  platform: gog
+  id: 1436866438
+  banner: steam:346940
+  status: unsupported
+
+- name: "Shadowrun Returns"
+  platform: gog
+  id: 1207660413
+  banner: steam:234650
+  status: unsupported
+
+- name: "Shadowrun: Dragonfall - Director's Cut"
+  platform: gog
+  id: 1207660913
+  banner: steam:300550
+  status: unsupported
+
+- name: "Shantae and the Pirate's Curse"
+  platform: gog
+  id: 1456141287
+  banner: steam:345820
+  status: unsupported
+
+- name: "Syberia"
+  platform: gog
+  id: 1207658848
+  banner: steam:46500
+  status: unsupported
+
+- name: "Syberia II"
+  platform: gog
+  id: 1207658849
+  banner: steam:46510
+  status: unsupported
+
+- name: "The Elder Scrolls II: Daggerfall"
+  platform: gog
+  id: 1435829353
+  banner: steam:1812390
+  status: unsupported
+
+- name: "The Elder Scrolls: Arena"
+  platform: gog
+  id: 1435828982
+  banner: steam:1812290
+  status: unsupported
+
+- name: "The Night of the Rabbit"
+  platform: gog
+  id: 1207659218
+  banner: steam:230820
+  status: unsupported
+
+- name: "The Supper"
+  platform: gog
+  id: 1481527544
+  banner: steam:1171370
+  status: unsupported
+
+- name: "The Whisperer"
+  platform: gog
+  id: 1426240474
+  banner: steam:1697940
+  status: unsupported
+
+- name: "The Witcher: Enhanced Edition"
+  platform: gog
+  id: 1207658924
+  banner: steam:20900
+  status: unsupported
+
+- name: "Venetica - Gold Edition"
+  platform: gog
+  id: 1345182195
+  banner: steam:338140
+  status: unsupported
+
+- name: "Worms Revolution Gold Edition"
+  platform: gog
+  id: 1421309312
+  banner: steam:200170
```

Comment:
The Steam version does not appear to be the gold edition.

Steamgriddb has some artwork for the gold edition:
https://cdn2.steamgriddb.com/grid/729fe11bfd2eab32e7613461d2f5ea74.png
https://cdn2.steamgriddb.com/grid/12059c268a74f3db7ed26b87efe23461.png

Reply:
You're right. I missed this. Corrected.